src/Calendar.php

```diff
@@ -110,6 +110,15 @@ public function addEvent(EventInterface $event)
         return $this;
     }
 
+    public function addEvents(array $events)
+    {
+        $this->events = array_filter($events, function($event) {
```

Comment:
This is a replacement rather than an addition (if I've added a couple of events singularly and then do a bulk-add, the previously added events are wiped). I'd expect an `addEvents()` method to not wipe the previous, so could you change this to be additive?